lib/Imager/Filters.pod

```diff
@@ -808,6 +808,35 @@ For large sample images this is scaled down to the range 0 .. 1.
 
 =back
 
+=item rgb_difference()
+
+You can create a new image that is the difference between 2 other images.
+
+  my $diff = $img->rgb_difference(other=>$other_img);
+
+For each pixel in $img that is different to the pixel in $other_img,
+the arithmetic difference for the value of the pixel in $img from
+$other_img per color is given. Transparency is ignored.
```

Comment:
"Transparency is ignored." - this doesn't appear to be the case.  If the input images have alpha channels they are included in the difference.

Reply:
Darn, you're right. I've addressed this in my commit c22ce745825deb9f76bde2d6197cbce48626a3df. Since I did the three commits in a different order, this code got mangled twice. You may want to review a squashed version.